.circleci/config.yml

```diff
@@ -1,29 +1,17 @@
-version: 2
+version: 2.1
 jobs:
   build:
     docker:
-      - image: circleci/python:3.7.0
+      - image: cimg/python:3.12
     steps:
       - checkout
       - setup_remote_docker
       - run:
           name: Build container
-          command: docker build -t ds-python .
+          command: docker build --target test -t ds-python .
       - run:
           name: Verify build completed
           command: docker run ds-python /bin/bash -c "echo BUILDS OK"
       - run:
-          name: Check that scipy links to OpenBLAS
-          command: docker run ds-python python -c "from scipy.linalg import _fblas"
```

Comment:
Moving this and other tests to the python unitttest script I'm adding below for readability. In general, as long as we verify that the key packages can be imported, we should be good.